listener/upload_listener.py

```diff
@@ -447,6 +457,21 @@ def process_message(msg):  # pylint: disable=too-many-return-statements,too-many
                 return
             process_func = process_upload
         elif msg.topic == mqueue.EVENTS_TOPIC:
+            LOGGER.info(str(msg_dict))
+            if msg_dict == 'webapps-refreshing':
```

Comment:
it is too late when `webapps-refreshing` is received here, VMaaS starts refreshing imediately after it sends the message and engine can have several evaluations in progress already and they will fail

VMaaS should wait for confirmation from engine that processing is stopped/paused and after then start refreshing (or after some reasonable timeout in case confirmation is not received)

ugh, it's becoming a bit complex but it doesn't make sense to use this solution which works only partially

Reply:
I see I think it is not a problem to notify vmaas that processing is paused